torchaudio/compliance/kaldi.py

```diff
@@ -92,10 +107,10 @@ def _feature_window_function(window_type, window_size, blackman_coeff):
         # like hanning but goes to zero at edges
         return torch.hann_window(window_size, periodic=False).pow(0.85)
     elif window_type == RECTANGULAR:
-        return torch.ones(window_size, dtype=torch.get_default_dtype())
```

Comment:
why don't we force the torch default type?

Reply:
torchscript does not support calling `torch.get_default_dtype`. In any case, the `dtype` of `torch.tensor`, `torch.ones` and `torch.arange` is already set to `torch.get_default_dtype`, see [torch.tensor](https://pytorch.org/docs/stable/torch.html#torch.tensor), [torch.ones](https://pytorch.org/docs/stable/torch.html#torch.ones) and [torch.arange](https://pytorch.org/docs/stable/torch.html#torch.arange). Is that what you meant?

Reply:
(relates to pytorch/pytorch#30774)